main.tf

```diff
@@ -112,7 +120,7 @@ resource "aws_lambda_function" "ami_backup" {
 resource "aws_lambda_function" "ami_cleanup" {
   filename         = "${path.module}/ami_cleanup.zip"
   function_name    = "${module.label_cleanup.id}"
-  description      = "Cleanup old AMI backups"
+  description      = "Automatically removal AMI images of EC2 instance (remove AMI)"
```

Comment:
`Automatically remove AMIs that have expired`

Reply:
Modified